src/transformers/integrations.py

```diff
@@ -23,6 +23,8 @@
 import tempfile
 from pathlib import Path
 
+import torch
```

Comment:
This should be protected behind an `if is_torch_available()` (see failing tests).